adyenv6core/resources/adyenv6core-beans.xml

```diff
@@ -73,6 +75,8 @@
         <property name="terminalId" type="String" />
         <property name="adyenLastName" type="String" />
         <property name="adyenBillingAddress" type="de.hybris.platform.commercewebservicescommons.dto.user.AddressWsDTO"/>
+        <property name="adyenApplePayMerchantName" type="java.lang.String"/>
+        <property name="adyenApplePayMerchantIdentifier" type="java.lang.String"/>
     </bean>
 
     <bean class="de.hybris.platform.commercewebservicescommons.dto.order.AbstractOrderWsDTO">
```

Comment:
Please double check if fulfillment flow for applepay works without adding properties here.